core/src/main/scala/org/apache/spark/deploy/security/HadoopFSDelegationTokenProvider.scala

```diff
@@ -99,11 +100,24 @@ private[deploy] class HadoopFSDelegationTokenProvider
   private def fetchDelegationTokens(
       renewer: String,
       filesystems: Set[FileSystem],
-      creds: Credentials): Credentials = {
+      creds: Credentials,
+      hadoopConf: Configuration,
+      sparkConf: SparkConf): Credentials = {
+
+    // The hosts on which the file systems to be excluded from token renewal
+    val fsToExclude = sparkConf.get(KERBEROS_FILESYSTEM_RENEWAL_EXCLUDE)
+      .map(new Path(_).getFileSystem(hadoopConf).getUri.getHost)
+      .toSet
 
     filesystems.foreach { fs =>
-      logInfo(s"getting token for: $fs with renewer $renewer")
-      fs.addDelegationTokens(renewer, creds)
+      if (fsToExclude.contains(fs.getUri.getHost)) {
+        // RM skips renewing token with empty renewer
```

Comment:
Just to understand properly, does `RM` mean Resource Manager? Now the delegation token handling is not only done for Spark on yarn but also for other resource schedulers as well, so probably better to remove specific resource scheduler's term/details.

Reply:
Actually you remind me that I should document it this is only for YARN. As the PR description mentions, this is for YARN behavior. I am not sure if other resource scheduler follows the same behavior here.

Reply:
We only test it under YARN and verify it works to skip the renewal from YARN if we give an empty renewer. I'm not sure if other resource scheduler follows this behavior. So I can just document the config to mention it is known to work for YARN.

Reply:
Oh gosh, maybe I'm working on too many stuff in parallel and was watching bad TV here :)
Today morning I've put other tasks away and has taken the time and re-checked/re-tested the code and it has no influence on the initial obtain.
This can be resolved and I'm fine with the code as-is.